azure-functions-maven-plugin/README.md

```diff
@@ -204,6 +205,10 @@ Supported deployment methods are listed as below. The default value is **ZIP**.
 - ~~~MSDeploy~~~ (deprecated)
 - ~~~FTP~~~ (deprecated)
 
+### Add Proxies Configuration to Your Azure Function
+
+[Azure Functions proxies](https://docs.microsoft.com/en-us/azure/azure-functions/functions-proxies) can be used for things like URL rewriting. To include a `proxies.json` file in your deployment you need to add it to the `maven-resources-plugin` configuration in `pom.xml` alongside `host.json` and `local.settings.json`.
```

Comment:
@mjg123  Thanks a lot for your contribution! And could you please share an example of `proxies.json`? It could be more convenient to have an example here.

Reply:
Sure thing